osc-server/src/main/java/org/osc/core/broker/rest/client/openstack/openstack4j/Openstack4JNeutron.java

```diff
@@ -156,7 +155,7 @@ public String getNetworkPortRouterDeviceId(String projectId, String region, Port
     private Port getPortByMacAddress(String region, String macAddress) {
         getOs().useRegion(region);
         List<? extends Port> portList = getOs().networking().port().list(PortListOptions.create().macAddress(macAddress));
-        return (portList == null || portList.isEmpty()) ? null : portList.get(0);
+        return portList == null || portList.isEmpty() ? null : portList.get(0);
```

Comment:
usage of CollectionUtils.emptyIfNull  might make is compact?.

Reply:
we can replace by CollectionUtils.emptyIfNull(portList).isEmpty() but not sure it adds lot of value